.github/workflows/test.yml

```diff
@@ -68,3 +68,26 @@ jobs:
         run: yarn workspace @thunderstore/nextjs build
       - name: Test @thunderstore/nextjs
         run: yarn workspace @thunderstore/nextjs jest
+
+  deploy:
+    name: Trigger deploy
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: github.ref == 'refs/heads/master' && github.event_name != 'pull_request'
+    strategy:
+      matrix:
+        app: [nextjs]
+    steps:
+      - name: Call deployment workflow
+        run: |
+          curl \
+            -X POST \
+            -u "${{ secrets.DEPLOY_WORKFLOW_DISPATCH_USER }}:${{ secrets.DEPLOY_WORKFLOW_DISPATCH_PAT }}" \
```

Comment:
What does the "PAT" in `DEPLOY_WORKFLOW_DISPATCH_PAT` stand for? Something to do with password?

Reply:
Personal Access Token